activestorage/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Add support for `upload` options with `GCSService`
```

Comment:
Could use an example here. What would someone evaluating an upgrade to Active Storage 6.1 need/want to know about this?

Reply:
Done, also updated the ActiveStorage overview docs with examples as per the Amazon S3 section.